### DISCUSSION TITLE
ARCore Geospatial API

### DISCUSSION THREAD

**Discussion by yan-ftr** [CariusLars/ar_flutter_plugin#143] [Q&A]
I am new to AR, but am trying to build a Flutter APP with AR with this plug-in. I may need to use ARCore Geospatial API for my app. I am wondering will this plug-in package support the Geospatial API in the future?

**Comment by OshOEz** [comment]
Hi ! @yan-ftr ! It's been a while since you posted your question ! 
Did you find a way to implement the Geospatial API on flutter ? 

I'm starting to dig into VPS and I wondered the same question ! Would you mind sharing how your journey went ?
